lib/testing/src/lib/process-services-cloud/pages/task-header-cloud-component.page.ts

```diff
@@ -32,6 +33,8 @@ export class TaskHeaderCloudPage {
     idField = element.all(by.css('span[data-automation-id*="id"] span')).first();
     descriptionField = element(by.css('span[data-automation-id*="description"] span'));
     taskPropertyList = element(by.css('adf-cloud-task-header adf-card-view div[class="adf-property-list"]'));
+    descriptionEditIcon = element(by.css('mat-icon[data-automation-id*="edit-icon-description"]'));
```

Comment:
I suggest we declare just 1 locator, and take the task header cloud element name dynamically. This way we can check the assertions for the all the elements on the Task Header Cloud.